lol-frontend/src/config/types/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface Player {
+  position: Array<number>;
+}
+export interface Player {
+  position: number[];
+}
```

Comment:
I think this is where you were defining your PlayerState model. You just need one I would go with number[] but its up to you either should work